src/wp-includes/global-styles-and-settings.php

```diff
@@ -255,3 +287,14 @@ function ( $item ) {
 		}
 	}
 }
+
+/**
+ * Private function to clean the caches under the theme_json group.
+ *
+ * @since 6.2.0
+ * @access private
+ */
+function _wp_clean_theme_json_caches() {
```

Comment:
See #3556, this should not be private, and thus not start with an underscore.

Reply:
This chunk of code will need to be removed from this PR as it was covered in PR #3556, which has now been committed via https://core.trac.wordpress.org/changeset/55086. I'll see if I can rebase and update the PR 🤞